staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go

```diff
@@ -560,6 +560,11 @@ func (vs *VSphere) Instances() (cloudprovider.Instances, bool) {
 	return vs, true
 }
 
+// InstancesV2 returns an implementation of InstancesV2 for vSphere.
+func (vs *VSphere) InstancesV2() (cloudprovider.InstancesV2, bool) {
+	return vs, true
```

Comment:
set this to `false` and let Azure OWNERs flip this on?

Reply:
nice for this, would add InstancesV2 support for Azure.

Reply:
It's already added - all built-in providers have this already implemented. It's just a matter of changing this value here.